tests/e2e/users/add-user.specs.js

```diff
@@ -73,7 +78,7 @@ describe('Add user  : ', () => {
     usersPage.openAddUserModal();
     addUserModal.fillForm('', 'Not Saved', '%4wbbygxkgdwvdwT65');
     addUserModal.submit();
-    const errorMessageUserName = element.all(by.css('span.help-block.ng-binding')).get(0);
+    const errorMessageUserName=element.all(by.css('span.help-block.ng-binding')).get(0);
```

Comment:
Please move this to a page object. Also, relying on that `ng-binding` is less than ideal. This is a good place to use the `attrs.test-xxxxx` technique to add a test-specific attribute on the page so we can reliably get this element.